lib/generators/components/component_generator.rb

```diff
@@ -7,23 +7,27 @@ class ComponentGenerator < Rails::Generators::NamedBase
 
     source_root File.expand_path("../templates", __FILE__)
 
+    def filename
```

Comment:
This method should be marked private. Thor basically executes all public methods in sequence. Not that it would do any harm but still :)